pyscript.core/src/stdlib.js

```diff
@@ -8,6 +8,27 @@
 
 import pyscript from "./stdlib/pyscript.js";
 
+class Ignore extends Array {
+    #add = false;
+    #paths;
+    #array;
```

Comment:
Stupid question but what does the hash mean?

Reply:
private fields ... nothing ever can reach those in consumers' land .... it's also a way to avoid future possible clashing with new APIs (methods or fields) added to the base class, in this case the `Array` one.

Reply:
Ah cool thanks! Today I learned haha